db/migrate/20240221024839_remove_show_redesign_display_settings_from_affiliates.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveShowRedesignDisplaySettingsFromAffiliates < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :affiliates, :show_redesign_display_settings, :boolean
```

Comment:
Can we add in the `default: false` here so that, on rollback, we'd have the same state we started with.